bundles/org.eclipse.equinox.simpleconfigurator/src/org/eclipse/equinox/internal/simpleconfigurator/Activator.java

```diff
@@ -43,7 +43,7 @@
  * Otherwise, no uninstallation will not be done.
  */
 public class Activator implements BundleActivator {
-	public final static boolean DEBUG = false;
+	public final static boolean DEBUG = Boolean.getBoolean("equinox.simpleconfigurator.debug");
```

Comment:
I'm unsure why this doesn't use the `org.eclipse.osgi.service.debug.DebugOptions` service to get trace options for the simple configurator.  Probably because of the ill-fated dream to have p2 be able to run on other framework implementations.

Reply:
I could think about that tracing might not be available at this stage? But not completely sure, that way you can at least enable it quite easy when starting Eclipse from commandline.

Reply:
We can leave that to another issue